library/src/main/java/org/xmtp/android/library/SigningKey.kt

```diff
@@ -31,7 +32,8 @@ fun SigningKey.createIdentity(identity: PrivateKeyOuterClass.PrivateKey): Author
     val signatureClass = Signature.newBuilder().build()
     val signatureText = signatureClass.createIdentityText(key = slimKey.toByteArray())
     val digest = signatureClass.ethHash(message = signatureText)
-    val signature = sign(digest)
+    //TODO(elise): Updated to use signatureText so it would be human readable until we can reverse eth hash
+    val signature = runBlocking { sign(signatureText) } ?: throw XMTPException("Illegal signature")
```

Comment:
I noticed the signature message looked like random bytes because we `ethHash` it. I think we'll want to expose a way to un`ethHash` the bytearray in order to display a human readable message when signing if we want to use the digest @nplasterer 👀 . What do you think?